configure.py

```diff
@@ -1120,6 +1120,9 @@ def configure_node(o):
     o['variables']['debug_nghttp2'] = 'false'
 
   if options.experimental_quic:
+    if options.shared_openssl:
+      raise Exception('QUIC requires modified version of OpenSSL and cannot be'
+                      ' enabled with --shared_openssl.')
```

Comment:
Should this perhaps be `--shared-openssl` instead, as that is the name of the option passed on the command line?